readme.md

```diff
@@ -1,79 +1,52 @@
 
 # Welcome to the Stalin Sort Repo :clipboard:
 
-Stalin Sort in any language you like! What is Stalin Sort?
+## What is Stalin Sort?
 
-![poster](stalin-sort.jpeg)
-
-  
-
-## Want to help? :star:
-
-We're accepiting help from anyone! This repo is collaborative, so if you want to add a solution in a new language (check Wikipedia [Programming Language List](https://en.wikipedia.org/wiki/List_of_programming_languages)) or just improve some code that you think that can be improved, follow this steps:
-
-  
-
-### 1. Fork it :fork_and_knife:
-
-  
-
-You can get your own fork/copy of [Stalin Sort](https://github.com/gustavo-depaula/stalin-sort) by using the <a  href="https://github.com/Hacktoberfest-2018/Hello-world/new/master?readme=1#fork-destination-box"><kbd><b>Fork</b></kbd></a> button or clicking [this](https://github.com/gustavo-depaula/stalin-sort/new/master?readme=1#fork-destination-box).
-
-  
-
-[![Fork Button](https://help.github.com/assets/images/help/repository/fork_button.jpg)](https://github.com/gustavo-depaula/stalin-sort/new/master?readme=1#fork-destination-box)
-
-  
-
-### 2. Start coding! :rabbit2:
+### Introduction
 
-Once you have forked the repo, add your program in the language folder in
+Stalin Sort is an efficient sorting algorithm, serving as a systematic method for placing the elements of a random access file or an array in order. Stalin Sort is also know as the best sorting algorithm of all times becasue of it's AMAZING capacity of always ordering an array with an O(n) performance.
 
-main branch, if there is no language folder, make one, then add into it.
+### How it works?
 
-  
+It's simple, all you need to do is iterate through the array checking if its elements are in order, any element that itsn't in order you pull out, in another words, you send it to Gulag.
 
-For a PR to be approved and merged please note
+### Step-by-step example
 
-- we allow more than one solution for one language, so if you don't want to modify someones code, you can add a new file with your own solution. Have in mind that your solution must be completely unique, if it isn't consider modify the existing code.
+1. (1 2 5 3 5 7) -> (**1** 2 5 3 5 7) Here the algorithm stores the first of element of the array
+2. (**1** 2 5 3 5 7) -> (1 **2** 5 3 5 7) Now it will compare the stored element with the second one, if this is bigger than the stored, it replaces the stored element by this
+3. (1 **2** 5 3 5 7) -> (1 2 **5** 3 5 7) Repeats step 2nd
+4. (1 2 **5** 3 5 7) -> (1 2 **5** 5 7) How the 4th element is smaller then the 3rd one that is just eliminated
+5. (1 2 **5** 5 7) -> (1 2 5 **5** 7) Equal elements are preserved
+6. **(1 2 5 5 7)** Ordered array!
 
-- we allow maximum of 5000 additions (your PR will get close if its exceeds, but you can ask maintainer for reviewing this)
+### Pseudocode implementation
```

Comment:
i think it's better to cut this section out, as people will try to copy this as the only way to do stalin sort

Reply:
the step by step example is fine

Reply:
I don't think so, we've a lot of different implementations and pseudocode is a good way to explain fast and easy what the code needs to do